pkg/controller/migplan/migplan_controller.go

```diff
@@ -66,6 +66,33 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 		return err
 	}
 
+	// Watch for changes to MigCluster
+	err = c.Watch(&source.Kind{
```

Comment:
We already spoke about this, but this kind of `Watch` will generate events for `MigCluster` nsNames rather than `MigPlan` nsNames that this controller is able to reconcile.

See https://book.kubebuilder.io/beyond_basics/controller_watches.html

Reply:
Agreed.  This was replaced when rebased.